### PR TITLE
MBS-9525: retry fullexport rsync if it fails

### PR DESCRIPTION
**Pull request by mwiencek**
This has failed twice recently: https://community.metabrainz.org/t/mbdump-tar-bz2-missing-from-latest-full-export/333175/5

This just retries it forever, sleeping for 30 seconds between each attempt.

### CODE REVIEW

**Comment by yvanzo** [comment]
I am not comfortable with this infinite loop.
Would it be reasonable to limit the number of retries?
Or to double the pause duration after each retry?

**Comment by mwiencek** [comment]
I think it's a good idea. And I figured out how to create a `retry` function in bash so we don't have to repeat the logic in both places now.

File: admin/functions.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```
Comment (mwiencek) [inline admin/functions.sh]: AFAICT this file is not sourced from any script that doesn't use `bash`.

**Comment by mwiencek** [comment]
Oops, I accidentally committed with the delay I was testing with. I changed the initial delay back to 15 seconds (so the delay before the final attempt should be 4 minutes, I think).

**Pull request merged**